conftest.py

```diff
@@ -31,3 +33,21 @@ def pytest_unconfigure(config):
     import sys
 
     del sys._called_from_test
+
+
+@pytest.fixture(scope="function", autouse=True)
```

Comment:
The scope here should probably be `session` because currently it's called before every test case, which takes a lot of time.

Reply:
Addressing in https://github.com/aiidateam/aiida-cp2k/pull/228

Reply:
fixed, thanks for the hint @oschuett